3.6/release-notes-new-features36.md

```diff
@@ -130,6 +130,60 @@ for this is
 Additionally, ArangoDB 3.6 provides a new AQL function `DATE_ROUND` to bin a date/time 
 into a set of equal-distance buckets.
 
+### Subquery Splicing Optimization
+
+In earlier versions of ArangoDB, on every execution of a subquery the following
+happened for each input row:
+
+    * The subquery tree issues one initializeCursor cascade through all nodes
+    * The subquery node pulls rows until the subquery node is empty for this input
+
+On subqueries with many results per input row (10000 or more) the
+above steps were not too expensive and typically overshadowed other overhead.
+On subqueries with few results there was a serious performance impact.
+
+Subquery splicing inlines the execution of subqueries using an optimizer rule
+called `subquery-splicing`. Only suitable queries can be spliced.
```

Comment:
In [execution-and-performance-optimizer.md](https://github.com/arangodb/docs/blob/feature/aql-subquery-optimisation-36/3.6/aql/execution-and-performance-optimizer.md) the rule is called `splice-subqueries`?